TestVectorsAwsCryptographicMaterialProviders/Makefile

```diff
@@ -153,6 +172,19 @@ TRANSLATION_RECORD_PYTHON := \
     --translation-record ../AwsCryptographyPrimitives/runtimes/python/src/aws_cryptography_primitives/internaldafny/generated/dafny_src-py.dtr \
 	--translation-record ../AwsCryptographicMaterialProviders/runtimes/python/src/aws_cryptographic_material_providers/internaldafny/generated/dafny_src-py.dtr
 
+# Smithy-dafny generated shim needs a long term fix. 
+# Smithy-dafny generates polymorph code of MPL in test vectors.  
+# TODO: Remove these commands once smithy-dafny is fixed
+# SED parameter is set on SmithyDafnySedMakefile.mk in smithy-dafny
+polymorph_go:
+	find .. -name "shim.go" | xargs sed -i $(SED_PARAMETER) 's/(_static \*CompanionStruct_Default___)//g'
```

Comment:
Add a link to the broken windows CI sim.

Reply:
Done in this PR: https://github.com/aws/aws-cryptographic-material-providers-library/pull/1157 because I had to make PR to main